larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -525,12 +587,45 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
 
   if (kernel_type == KernelType::kRuyOptimized) {
     switch (conv_params->bitpacking_bitwidth) {
+// On Arm64, we don't yet support 8-bit bitpacked input or writing bitpacked
+// output.
+#if RUY_PLATFORM(ARM_64)
```

Comment:
I think on Arm64 we should let it fall-back to the C++ kernel. Then you can remove this entire `#if RUY_PLATFORM(ARM_64)` block.

I would suggest adding a line near here:
https://github.com/larq/compute-engine/blob/8e8da88a3ce997da4705f3713f282c9bde1154f0/larq_compute_engine/core/bgemm_impl_ruy.h#L74
with something like

```cpp
// For writing bitpacked output, fallback to the standard C++ kernel.
if (std::is_same<DstScalar, std::uint8_t>::value) {
  bgemm_runtime_path = ruy::Path::kStandardCpp;
}
```

Reply:
Nice, that's a good idea

Reply:
I do think this is a good idea but a first pass at it spews a lot of compiler errors (adding the three lines to `bgemm_impl_ruy.h` works fine, but removing the `#if RUY_PLATFORM(ARM_64)` breaks).